scripts/release_info.py

```diff
@@ -411,6 +513,156 @@ def main():
 
     python_utils.PRINT('Done. Summary file generated in ../release_summary.md')
 
+    while True:
+        python_utils.PRINT(
+            '******************************************************')
+        python_utils.PRINT(
+            'Please update ../release_summary.md to:\n'
+            '- have a correct changelog for '
+            'updating the CHANGELOG file\n'
+            '- have a correct list of new authors and contributors to '
+            'update AUTHORS, CONTRIBUTORS and Credits section in '
+            'about-page.directive.html\n'
+            'Confirm once you are done by entering y/ye/yes.\n')
+        answer = python_utils.INPUT().lower()
+        if answer in ['y', 'ye', 'yes']:
+            break
+
+    current_release_version = branch_name[len(
+        common.RELEASE_BRANCH_NAME_PREFIX):]
+    current_date = datetime.date.today().strftime('%d %b %Y')
+    source_branch = 'develop'
+    target_branch = 'update-changelog-for-releasev%s' % current_release_version
+    # This complete code block is wrapped in try except since in case of
+    # an exception, we should ensure that the changes made to AUTHORS,
+    # CONTRIBUTORS, CHANGELOG and about-page are reverted and the branch
+    # created is deleted. This is required to avoid errors when running
+    # the script again.
+    try:
+        with python_utils.open_file(
+            '../release_summary.md', 'r') as release_summary_file:
```

Comment:
if this filename is used in multiple place then let's create a constant var.

Reply:
Done.